setup.sh

```diff
@@ -0,0 +1,99 @@
+#! /bin/sh
```

Comment:
```suggestion
#! /bin/bash
```

This makes invoking `./setup.sh` work on my machine. Otherwise it tries to use `sh` which doesn't support some of the syntax used

Reply:
Thank you!